spec/features/card_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe Card, :type => :feature do
+  let!(:card) { create(:card) }
+
+  it "card checking" do
+    visit new_review_path
+    within(".simple_form") do
+      fill_in 'review_translated_text', :with => 'deer'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.